### PR TITLE
fix: abnormal style due to long title of recent posts in the dashboard

### PR DESCRIPTION
**Pull request by Aanko**
Fixes halo-dev/halo#1836

### CODE REVIEW

File: src/views/dashboard/Dashboard.vue [hunk 1]
```diff
@@ -62,18 +62,13 @@
                           {{ item.title }}
                         </a>
 
-                        <a-button
-                          v-else-if="item.status === 'DRAFT'"
-                          class="!p-0"
-                          type="link"
-                          @click="handlePostPreview(item.id)"
-                        >
+                        <a v-else-if="item.status === 'DRAFT'" @click="handlePostPreview(item.id)">
```
Comment (ruibaby) [inline src/views/dashboard/Dashboard.vue:65]:
```suggestion
                        <a herf="javascript:void(0)" v-else-if="item.status === 'DRAFT'" @click="handlePostPreview(item.id)">
```

File: src/views/dashboard/Dashboard.vue [hunk 2]
```diff
@@ -62,18 +62,13 @@
...
                           {{ item.title }}
-                        </a-button>
+                        </a>
 
-                        <a-button v-else-if="item.status === 'RECYCLE'" class="!p-0" disabled type="link">
+                        <a v-else-if="item.status === 'RECYCLE'" disabled>
```
Comment (ruibaby) [inline src/views/dashboard/Dashboard.vue:69]:
```suggestion
                        <a  herf="javascript:void(0)" v-else-if="item.status === 'RECYCLE'" disabled>
```

**Review by ruibaby — CHANGES_REQUESTED** [review]

**Comment by ruibaby** [comment]
/retitle fix: abnormal style due to long title of recent posts in the dashboard

**Comment by ruibaby** [comment]
你好 @Aanko ，请按照提示 sign 一下我们的 CLA。https://github.com/halo-dev/halo-admin/pull/545#issuecomment-1094574738

**Comment by guqing** [comment]
Hi @Aanko 需要签一下证书
image

File: src/views/dashboard/Dashboard.vue [hunk 3]
```diff
@@ -62,18 +62,13 @@
...
-                          type="link"
-                          @click="handlePostPreview(item.id)"
-                        >
+                        <a herf="javascript:void(0)" v-else-if="item.status === 'DRAFT'" @click="handlePostPreview(item.id)">
                           {{ item.title }}
-                        </a-button>
+                        </a>
 
-                        <a-button v-else-if="item.status === 'RECYCLE'" class="!p-0" disabled type="link">
+                        <a  herf="javascript:void(0)" v-else-if="item.status === 'RECYCLE'" disabled>
```
Comment (ruibaby) [inline src/views/dashboard/Dashboard.vue:69]:
```suggestion
                        <a herf="javascript:void(0)" v-else-if="item.status === 'RECYCLE'" disabled>
```

**Review by ruibaby — CHANGES_REQUESTED** [review]

**Review by ruibaby — APPROVED** [review]
/lgtm
/approve

**Review by guqing — APPROVED** [review]
/lgtm

**Review by QuentinHsu — APPROVED** [review]

**Pull request merged**